### ISSUE TITLE
Official Tracing WASM Binaries

### ISSUE THREAD

**Issue by insipx** [paritytech/substrate-archive#208]
So far WASM binaries for Polkadot/Kusama are kept in this repository and updated along with Polkadot. it might be worth making it more official and as part of the polkadot release cycle to make upgrading easier

**Comment by s3krit** [comment]
Happy to help in any way I can on this. Assigned myself so I don't lose track of it, but it'll take a little while for me to internalise the rather large PR this issue stemmed from ;)

**Comment by dvdplm** [comment]
> Happy to help in any way I can on this. Assigned myself so I don't lose track of it, but it'll take a little while for me to internalise the rather large PR this issue stemmed from ;)

Yeah no worries at all, we’ll come looking for you when the time is ripe. :)
